iwyu_path_util.cc

```diff
@@ -109,8 +109,7 @@ string CanonicalizeHeaderSearchPath(const string& path) {
 
 string GetCanonicalName(string file_path) {
   // Get rid of any <> and "" in case file_path is really an #include line.
-  StripLeft(&file_path, "\"") || StripLeft(&file_path, "<");
-  StripRight(&file_path, "\"") || StripRight(&file_path, ">");
+  file_path = RemoveQuotes(file_path);
```

Comment:
Can you call this `StripQuotes` instead, to follow the Python terminology already used?